functorch/functorch/_src/eager_transforms.py

```diff
@@ -260,13 +260,23 @@ def vjp(func: Callable, *primals, has_aux: bool = False):
         outer one. This is because ``vjp`` is a "function transform": its result
         should not depend on the result of a context manager outside of ``f``.
     """
+    return _vjp_with_argnums(func, *primals, has_aux=has_aux)
+
+
+# only to be used with jacrev to allow for all passed variables to be wrapped correctly, even if they don't participate
+# in autograd
+def _vjp_with_argnums(func: Callable, *primals, argnums: Optional[argnums_t] = None, has_aux: bool = False):
```

Comment:
Nit: we should comment somewhere that `argnums = None` means everything gets differentiated

Reply:
Moved the comment below _vjp_with_argnums and noted what `argnums=None` means along with what it means when there are argnums (especially wrt what the returned `vjp_fn` returns)